parley/src/layout/cursor.rs

```diff
@@ -87,50 +84,58 @@ impl Cursor {
     }
 
     /// Creates a new cursor for the specified layout and text position.
-    pub fn from_position<B: Brush>(
-        layout: &Layout<B>,
-        mut position: usize,
-        is_leading: bool,
-    ) -> Self {
+    pub fn from_byte_index<B: Brush>(layout: &Layout<B>, mut index: usize) -> Self {
```

Comment:
My understanding from reading the code is that, if the byte index you give is in the middle of a grapheme, the cursor returned will basically store a range from the beginning to the end of that grapheme. This property is used by prev_logical where you request the grapheme that contains the byte before the beginning of the current grapheme.

I think that behavior should be described in the doc comment.